Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pipeline {
     agent any
 
     stages {
-        stage('Gradle') {
+        stage('Unit Tests') {
```

Comment:
Is this a good name? What if we add integration tests and hook it into `./gradlew test` later?